src/main/java/com/tfflabs/soundfoundry/controllers/EventsController.java

```diff
@@ -0,0 +1,38 @@
+package com.tfflabs.soundfoundry.controllers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.messaging.handler.annotation.SendTo;
+import org.springframework.stereotype.Controller;
+import com.tfflabs.soundfoundry.dto.Event;
+import com.tfflabs.soundfoundry.services.RoomService;
+
+@Controller
+public class EventsController {
+
+	@Autowired
+	private RoomService roomService;
+	
+	private Logger log = LoggerFactory.getLogger(EventsController.class);
+	
+	@MessageMapping("/events")
+	@SendTo("/topic/events")
+	public Event processEvent(Event event){
+		log.info("Entering this shit");
+		log.debug("Event: {}", event);
+		switch(event.getType()){
+		
+		case ADDTRACK:
+			//Track track = new Track();

```

Comment:
Remove comments?

Reply:
Yeah, that was only to talk with you guys about the possibilities....